File: src/main/java/com/example/demo/DemoApplication.java
package com.example.demo;

import com.example.demo.scala.Pet;

//@SpringBootApplication
public class DemoApplication {

    public static void main(String[] args) {

//        SpringApplication.run(DemoApplication.class, args);
        Pet pet = new Pet();
        pet.print();
    }

}

File: src/main/java/com/example/demo/DemoApplication.kt
package com.example.demo

import com.example.demo.scala.Pet

//@SpringBootApplication
object DemoApplication {

    @JvmStatic
    fun main(args: Array<String>) {

        //        SpringApplication.run(DemoApplication.class, args);
        val pet = Pet()
        pet.print()
    }

}
